README.md

```diff
@@ -5,17 +5,20 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/decred/dcrdata)](https://goreportcard.com/report/github.com/decred/dcrdata)
 [![ISC License](https://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
 
-The dcrdata repository is a collection of golang packages and apps for [Decred](https://www.decred.org/) data collection, storage, and presentation.
+The dcrdata repository is a collection of go packages and apps for [Decred](https://www.decred.org/) data collection, storage, and presentation.
```

Comment:
Since you're going to be making another edit, note that the language is called "Go" while the tools is "go".